### PR TITLE
Task#68799: Updated backend listorder

### PR DESCRIPTION
**Pull request by dangvm-0369**
## Related Tickets:
- [Task#68799](https://edu-redmine.sun-asterisk.vn/issues/68799)

## WHAT this PR does?
Update BE List Order

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by dangvm-0369** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -15,11 +17,8 @@ class OrderController extends Controller
      */
     public function index()
     {
-        $orders = Order::with(['products', 'customer'])->whereDate('created_at', now()->toDateString());
-
         return Inertia::render('orders/Index.jsx', [
-            'orders' => $orders->limit(config('app.limit_table_records'))->orderByDesc('created_at')->get(),
-            'max_ordinal' => $orders->count(),
+            [ 'orders' => $this->transformOrder(), ]
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]:
```suggestion
            ['orders' => $this->transformOrder()],
```

**Comment by dangvm-0369** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 2]
```diff
@@ -15,11 +17,8 @@ class OrderController extends Controller
      */
     public function index()
     {
-        $orders = Order::with(['products', 'customer'])->whereDate('created_at', now()->toDateString());
-
         return Inertia::render('orders/Index.jsx', [
-            'orders' => $orders->limit(config('app.limit_table_records'))->orderByDesc('created_at')->get(),
-            'max_ordinal' => $orders->count(),
+            ['orders' => $this->transformOrder(),]
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]:
```suggestion
            ['orders' => $this->transformOrder()],
```

**Comment by dangvm-0369** [comment]
ready

**Pull request merged**